scripts/dev/dockerfiles/Dockerfile.readiness

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.13-alpine as builder
```

Comment:
should we bump the go version here as well?

Reply:
yes I think there's no harm doing that